lib/java/com/google/android/material/appbar/CollapsingToolbarLayout.java

```diff
@@ -178,6 +180,13 @@ public CollapsingToolbarLayout(@NonNull Context context, @Nullable AttributeSet
             R.styleable.CollapsingToolbarLayout_collapsedTitleGravity,
             GravityCompat.START | Gravity.CENTER_VERTICAL));
 
+    if (a.hasValue(R.styleable.CollapsingToolbarLayout_titleTextAlignment)) {
+      int id = a.getInt(
+          R.styleable.CollapsingToolbarLayout_titleTextAlignment, 0);
+      Alignment alignment = Alignment.values()[id];
```

Comment:
It's better to explicitly create a conversion method here to prevent the dependency on enum orders, which can be fragile.